src/components/LoginCallback.vue

```diff
@@ -11,12 +11,18 @@
  */
 
 
-<script>
-export default {
+<script lang="ts">
+import Vue from 'vue'
+
+export default Vue.extend({
   name: 'LoginCallback',
   async beforeMount () {
     await this.$auth.handleLoginRedirect()
   },
-  render () {}
-}
+  render() { 
+    // typescript not allow empty render function, here's the workaround for renderless component
```

Comment:
Not for this PR, but is this a sign we're doing something less than ideal?

Reply:
While develop this PR, I personally feel Vue is still not ready for typescript. Lots of tricks (not clearly documented) need to be handled in both the SDK and the application code.

The good thing about the change here is that it's transparent to the consumer, and we can make change freely when a better solution is available.

Reply:
Please collect those tricks so we can create tickets to clean things up, be that in documentation or programmatic approach.

Reply:
Comments in code should be good for now as we don't know will it be officially addressed and when.

Reply:
oh, you mean Vue, not okta-vue, when you said "not ready for typescript".   I misunderstood.